openedx/core/djangoapps/courseware_api/serializers.py

```diff
@@ -106,7 +106,8 @@ class CourseInfoSerializer(serializers.Serializer):  # pylint: disable=abstract-
     original_user_is_staff = serializers.BooleanField()
     can_view_legacy_courseware = serializers.BooleanField()
     is_staff = serializers.BooleanField()
-    can_load_courseware = serializers.DictField()
+    course_access = serializers.DictField()
+    can_load_courseware = serializers.DictField(source='course_access')  # can be removed once MFE updates
```

Comment:
I've renamed the access field to be `course_access` for both course home and courseware. Here, I'm keeping the old name for a tiny bit - as soon as the MFE is deployed, I can remove this old `can_load_courseware` name. For context:

**Before**
Courseware: `can_load_courseware` - dictionary of access error info, which includes whether the MFE was enabled as a possible error state
Course Home: `can_load_courseware` - bool for whether the courseware MFE was enabled

**After**
Courseware: `course_access` - dictionary of access error info, which includes whether the MFE was enabled as a possible error state (just the name changed here) 
Course Home:
`can_load_courseware` - same as before - a bool for whether the courseware MFE was enabled.
`course_access` - new dictionary of access error info (but no MFE enabled info)